src/process_utility.c

```diff
@@ -1011,6 +1011,15 @@ process_truncate(ProcessUtilityArgs *args)
 		}
 		else
 		{
+			Chunk *chunk = ts_chunk_get_by_relid(relid, false);
+			if (chunk != NULL)
+			{
+				ereport(ERROR,
+						(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+						 errmsg("truncating chunks not supported"),
```

Comment:
Actually, if I may nitpick: should add "is" to be similar to standard PostgreSQL messages about things not supported:

```suggestion
						 errmsg("truncating chunks is not supported"),
```